app/assets/stylesheets/finder_frontend.scss

```diff
@@ -157,6 +157,10 @@
     width:14px;
   }
 
+  .select-filter {
+    margin-bottom: $gutter-one-third;
```

Comment:
I'm going to remove this because the autocomplete component should come with it's own default bottom margin and it doesn't, which I'm correcting in https://github.com/alphagov/govuk_publishing_components/pull/680